website/blog/2019-03-07-releasing-react-native-059.md

```diff
@@ -0,0 +1,74 @@
+> * [useHooks.com](https://usehooks.com/) showcases community-maintained Hooks recipes and demos.
+
+Be sure to give this a try in your apps. We hope that you find the reuse as exciting as we do.
+
+## 📱 Updated JSC means performance gains and 64-bit support on Android
+
+React Native uses JSC ([JavaScriptCore](https://webkit.org/)) to power your application. JSC on Android was a few years old, which meant that a lot of modern JavaScript features weren't supported. Even worse, it performed poorly compared iOS's modern JSC. With this release, that all changes.
+
+Thanks to some awesome work by [@DanielZlotin](https://github.com/danielzlotin), JSC has caught up with the past few years. This brings with it 64-bit support, modern JavaScript support, and [big performance improvements](https://github.com/react-native-community/jsc-android-buildscripts/tree/master/measure). Kudos to Daniel for also making this a maintainable process now so that we can take advantage of future WebKit improvements without so much legwork.
+
+## 💨 Faster app launches with inline requires
+
+We want to help people have performant React Native apps by default and are working to bring Facebook's optimizations to the community. Applications load resources as needed rather than slowing down launch. This feature is called "inline requires", as it lets Metro identify components to be lazy loaded. Apps with a deep and varied component architecture will see the most improvement.
+
+![source of the `metro.config.js` file in the 0.59 template, demonstrating where to enable `inlineRequires`](/react-native/blog/assets/inline-requires.png)
+
+We need the community to let us know how it works before we turn it on by default. When you upgrade to 0.59, there will be a new `metro.config.js` file; flip the options to true and give us [your feedback](https://twitter.com/hashtag/inline-requires)! Read more about inline requires [in the performance docs](https://facebook.github.io/react-native/docs/0.56/performance#inline-requires) to benchmark your app.
+
+## 🚅 Lean core is underway
+
+React Native is a large and complex project with a complicated repository. This makes the codebase less approachable to contributors, difficult to test, and bloated as a dev dependency. [Lean Core](https://github.com/react-native-community/discussions-and-proposals/issues/6) is our effort to address these issues by migrating code to separate libraries for better management. The past few releases have seen the first steps of this, but [let's get serious](https://www.youtube.com/watch?v=FMLKb4or8yg).
+
+You may notice that additional components are now officially deprecated. This is great news, as there are now owners for these features actively maintaining them. Heed the warning messages and migrate to the new libraries for these features, because in 0.60 they will be removed. Below is a table indicating the component, its status, and where you may migrate your use to.
```

Comment:
```suggestion
You may notice that additional components are now officially deprecated. This is great news, as there are now owners for these features actively maintaining them. Heed the warning messages and migrate to the new libraries for these features, because in they will be removed in a future release. Below is a table indicating the component, its status, and where you may migrate your use to.
```

Might not want to commit to removing in a specific version in case we decide later to push removing them